### ISSUE TITLE
Failed to run socket reporter child process

### ISSUE THREAD

**Issue by samschurter** [SocketDev/vscode-socket-security#4]
Installed v1.0.0 on VSCode 1.75.0

VSCode Info:
```
Version: 1.75.0 (Universal)
Commit: e2816fe719a4026ffa1ee0189dc89bdfdbafb164
Date: 2023-02-01T15:24:42.903Z
Electron: 19.1.9
Chromium: 102.0.5005.194
Node.js: 16.14.2
V8: 10.2.154.23-electron.0
OS: Darwin arm64 22.3.0
Sandboxed: No
```

OS Info: 
```
Chip: Apple M1
Memory: 8GB
macOS: 13.2
```

Window log output:
```
2023-02-07 12:56:03.105 [error] [Extension Host] ERROR in Reporting Failed to run socket reporter child process
2023-02-07 12:59:31.781 [warning] Via 'product.json#extensionEnabledApiProposals' extension 'github.vscode-pull-request-github' wants API proposal 'commentsResolvedState' but that proposal DOES NOT EXIST. Likely, the proposal has been finalized (check 'vscode.d.ts') or was abandoned.
2023-02-07 12:59:34.437 [error] [Extension Host] ERROR in Reporting Failed to run socket reporter child process
2023-02-07 12:59:34.631 [info] [perf] Render performance baseline is 18ms
2023-02-07 13:06:25.726 [warning] Via 'product.json#extensionEnabledApiProposals' extension 'github.vscode-pull-request-github' wants API proposal 'commentsResolvedState' but that proposal DOES NOT EXIST. Likely, the proposal has been finalized (check 'vscode.d.ts') or was abandoned.
2023-02-07 13:06:28.526 [info] [perf] Render performance baseline is 17ms
2023-02-07 13:06:28.715 [error] [Extension Host] ERROR in Reporting Failed to run socket reporter child process
```

Uninstall/re-install and close/re-open VSCode did not solve the issue.

**Comment by samschurter** [comment]
Further information, this is happening when opening VS Code with a specific project. I opened a different project and the extension loaded fine. I can share whatever configuration or package files you think would be useful

**Comment by bmeck** [comment]
Looking now; I'm wondering is the project a remote workspace. That error is most likely from being unable to spawn a child process inside the extension itself, we are migrating to a non-child spawning version in a while but for now I'd check permissions on that project being allowed to run trusted scripts and if it is remote it will not work for now.

**Comment by samschurter** [comment]
It's actually a local workspace. All of my projects are local, and all inside a single parent directory with the parent folder and child folders trusted in VS Code. I double checked that and it's all good. 

I also usually have a lot of other extensions running, but disabling everything except Socket, Gitlens, Github Copilot, and ESLint did not solve the issue. 

Could the repo layout and multiple `package.json` files be crashing it? Our folder layout in the project is something like this:
- root
  - cloudFunctions
    - functionA
      - package.json
    - functionB
      - package.json
  - frontend
    - package.json

**Comment by samschurter** [comment]
Hey @bmeck, I sent you an email with the repo that's causing trouble with all of the code etc. deleted. It still causes the child process to fail.

**Comment by bmeck** [comment]
This should be fixed with `1.0.2` but let me know if it doesn't work.

**Comment by samschurter** [comment]
`1.0.2` fixed it, thanks!
